package.json

```diff
@@ -14,10 +14,10 @@
 		"npm": "^10.5.0"
 	},
 	"scripts": {
-		"lint:js": "eslint 'Resources/Public/**/*.js'",
-		"lint:js:fix": "eslint 'Resources/Public/**/*.js' --quiet --fix",
-		"lint:style": "stylelint Resources/Public/**/*.css",
-		"lint:style:fix": "stylelint Resources/Public/**/*.css --fix"
+		"ci:lint:js": "eslint 'Resources/Public/**/*.js'",
+		"fix:lint:js": "eslint 'Resources/Public/**/*.js' --quiet --fix",
+		"ci:lint:style": "stylelint Resources/Public/**/*.css",
+		"fix:lint:style": "stylelint Resources/Public/**/*.css --fix"
```

Comment:
Please let's move the renaming to a separate pre-patch PR to follow the "only one thing per PR" principle. :-)